spec/features/map/access_map_page_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe "Map Page", type: :feature do
+  it "is accessible" do
+    visit map_path
+  end
+
+  it "is accessible through navbar" do
```

Comment:
as this doesnt visit the map path, I would suggest naming it "is linked to from navbar"